ci.nix

```diff
@@ -1,15 +1,28 @@
 { pkgs ? import <nixpkgs> {} }:
 
 let
-  default = import ./default.nix {};
-
-  buildInputs = builtins.attrValues {
-    inherit (pkgs) gnumake which;
-
-    inherit (default) purs pulp purp psc-package dhall-simple spago pscid spago2nix purty zephyr;
-  };
+  inputsFromPkgs = with pkgs; [
+    gnumake
+    which
+  ];
 
+  default = import ./default.nix {};
```

Comment:
please, i do not want to use this method

Reply:
So use the commented out line below? (`inputsFromDefault = with (import ./default.nix {}); [`) Sorry, not sure what you had in mind.

Reply:
i do not want to use the changes in this file, and i especially do not ever want `with` in my code

Reply:
Removed. (Posting this here as a reminder to myself: [Nix anti-patterns: `with attrset; ...` expression](https://nix.dev/anti-patterns/language.html#with-attrset-expression))